From 8958cba832ae01bb953ee09f12185fde0ef5f36a Mon Sep 17 00:00:00 2001
From: Robert Chiras <robert.chiras@nxp.com>
Date: Tue, 12 Jun 2018 14:28:56 +0300
Subject: [PATCH 3977/5242] MLK-18543-3: arm64: dts: imx8qxp: Remove dsi&ldb
 phy power domains

commit  67e526fe26b700ce4792e64bda03bcadf00c8c65 from
https://source.codeaurora.org/external/imx/linux-imx.git

The MIPI-DSI and LVDS PHY power domains are managed by their bridge
drivers, there is no need to have power domains for the phys also.

Signed-off-by: Robert Chiras <robert.chiras@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi |    4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi
index df2d716..a18f114 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi
@@ -1706,7 +1706,6 @@
 		#size-cells = <0>;
 		compatible = "mixel,imx8qxp-mipi-dsi-phy";
 		reg = <0x0 0x56228300 0x0 0x100>;
-		power-domains = <&pd_mipi_dsi0>;
 		#phy-cells = <0>;
 		status = "disabled";
 	};
@@ -1781,7 +1780,6 @@
 		#phy-cells = <0>;
 		clocks = <&clk IMX8QXP_MIPI0_LVDS_PHY_CLK>;
 		clock-names = "phy";
-		power-domains = <&pd_mipi_dsi_0_lvds>;
 		status = "disabled";
 	};
 
@@ -1870,7 +1868,6 @@
 		#size-cells = <0>;
 		compatible = "mixel,imx8qxp-mipi-dsi-phy";
 		reg = <0x0 0x56248300 0x0 0x100>;
-		power-domains = <&pd_mipi_dsi1>;
 		#phy-cells = <0>;
 		status = "disabled";
 	};
@@ -1945,7 +1942,6 @@
 		#phy-cells = <0>;
 		clocks = <&clk IMX8QXP_MIPI1_LVDS_PHY_CLK>;
 		clock-names = "phy";
-		power-domains = <&pd_mipi_dsi_1_lvds>;
 		status = "disabled";
 	};
 
-- 
1.7.9.5

